### PR TITLE
User meetings view

### PR DESCRIPTION
**Pull request by wdvf**

### CODE REVIEW

File: src/main/java/org/jala/efeeder/user/UserMeetingsCommand.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jala.efeeder.user;
```
Comment (MirkoTV) [inline src/main/java/org/jala/efeeder/user/UserMeetingsCommand.java:1]: please remove this header

File: src/main/webapp/WEB-INF/user/history.jsp [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<%@page contentType="text/html" pageEncoding="UTF-8"%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
```
Comment (MirkoTV) [inline src/main/webapp/WEB-INF/user/history.jsp:2]: Could you see if this taglib is really necessary in this specific file ? (<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>). if yes ignore this please.

File: src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java [hunk 3]
```diff
@@ -48,6 +48,10 @@ public FoodMeeting(int id, String name, String imageLink, Timestamp eventDate, U
 		this(id, name, imageLink, FoodMeetingStatus.Voting, eventDate, eventDate, eventDate, eventDate, null, userOwner);
 	}
 
+	public FoodMeeting(int id, String name, String imageLink, FoodMeetingStatus status, Timestamp eventDate, User userOwner) {
+		this(id, name, imageLink, status, eventDate, eventDate, eventDate, eventDate, null, userOwner);
```
Comment (MirkoTV) [inline src/main/java/org/jala/efeeder/foodmeeting/FoodMeeting.java:52]: If I understand this, here we are creating the FoodMeetings without payment time. Is that correct? if so, is there a reason to do that?

**Review by MirkoTV — CHANGES_REQUESTED** [review]

**Comment by wdvf** [comment]
@MirkoTV please check it again

**Review by MirkoTV — APPROVED** [review]

**Pull request merged**